kythe/rust/indexer/cargo/remote/bstr-0.2.13.BUILD

```diff
@@ -0,0 +1,63 @@
+"""
+cargo-raze crate build file.
+
+DO NOT EDIT! Replaced on runs of cargo-raze
+"""
+package(default_visibility = [
+  # Public for visibility by "@raze__crate__version//" targets.
+  #
+  # Prefer access through "//kythe/rust/indexer/cargo", which limits external
+  # visibility to explicit Cargo.toml dependencies.
+  "//visibility:public",
+])
+
+licenses([
+  "restricted", # "MIT OR Apache-2.0"
```

Comment:
Do you know why this is `restricted`? Going by go/thirdpartylicenses#notice I'd think this would be `notice` instead.

Reply:
I'm not sure. This is automatically set by `cargo-raze`

Reply:
Does cargo-raze not understand disjunctive licenses? In particular, this package has the license MIT *OR* Apache-2.0, both of which are `notice`-class licenses.

Reply:
@Arm1stice found where raze parses license strings: https://github.com/google/cargo-raze/blob/a1e03583fa4bc64937d8d447f5b954c04b8b3ffb/impl/src/license.rs#L46

Seems like it understands `/` but not `OR`.

Reply:
I'm for now I'll change this manually but I will submit a PR to cargo-raze to try and get this fixed.